sale_restricted_qty/__manifest__.py

```diff
@@ -1,24 +1,21 @@
 # Copyright 2019 Akretion (<http://www.akretion.com>)
+# Copyright 2024 CorporateHub
```

Comment:
Why this ?

Reply:
Because file was altered during migration?

Reply:
> Because file was altered during migration?

Which one? I don't see this in any version.

Reply:
Sorry, I don't follow. The `__manifest__.py` itself as well as bunch of others? https://github.com/OCA/sale-workflow/pull/2992/commits/bdef9490feb16762954183c82d173a33ce265197#diff-693caeb9136865eee2be7718a59a9d3360c8a38f48ae3a6bc510e88c176a2df3